GeoPackage - Field domain added to Browser (#9276)
qgis/QGIS-Documentation: docs/user_manual/managing_data_source/opening_data.rst

.. index:: ArcInfo Binary Coverage, Tiger Format, UK National Transfer Format
.. index:: US Census Bureau

.. note::

 When loading vector layers into QGIS, fields with |sourceFields| :guilabel:`Field Domains`
 (such as those defined in a GeoPackage or ESRI File Geodatabase) are automatically
 detected. These domains are database-level constraints, meaning they are enforced
 by the database itself and apply across different applications, not just QGIS.

>  This differs from the QGIS-only **Value Maps** widget, which is saved within QGIS project styles
>  and is not enforced outside QGIS.

Reviewer: actually, you have:
* "coded values" automatically detected by "value maps" widget
* "range domains" automatically detected by "range" widget
* and "glob" ... I couldn't find the syntax to better experiment if it is recognized by something.

```suggestion
 This differs from the QGIS-only widgets, which are saved within QGIS project styles
 and are not enforced outside QGIS.
```
or just remove to focus on the feature, at the db level?
